### PR TITLE
add bdist_rpm to build on top of latest release

### PR DESCRIPTION
**Pull request by wdpypere**

### CODE REVIEW

File: pakiti.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Summary:	Patching status monitoring tool
 Name:		pakiti-client
-Version:	X
-Release:	1%{?dist}
+Version:	3.0.3
+Release:	1
```
Comment (stdweird) [inline pakiti.spec]: needs a `.%{gittag}`

Comment (stdweird) [reply pakiti.spec]: well, `.%{gittag}%{?dist}.ug`

**Comment by wdpypere** [comment]
@stdweird, now it works

File: pakiti-client.spec [hunk 2]
```diff
@@ -1,11 +1,12 @@
 Summary:	Patching status monitoring tool
 Name:		pakiti-client
-Version:	X
-Release:	1%{?dist}
+Version:	3.0.3
+%global rel	1
+Release:	%{rel}%{gittag}.ug
```
Comment (stdweird) [inline pakiti-client.spec]: what is the full final rpm version with these?

Comment (wdpypere) [reply pakiti-client.spec]: ```RPMS/noarch/pakiti-client-3.0.3-11650987211.e50c991.ug.noarch.rpm```

Comment (stdweird) [reply pakiti-client.spec]: thought as much. there is a missing `.` between `%rel ` and `%gittag`. if you have made the rpm and put it in the repos, you need to bump the version to 3.0.4

Comment (wdpypere) [reply pakiti-client.spec]: ah, then the spunk talamini one is wrong as well :) . Dot fixed. I had not uploaded an rpm yet.

Comment (stdweird) [reply pakiti-client.spec]: https://github.com/hpcugent/slurm-spank-talamini/blob/master/slurm-spank-talamini.spec#L7 looks ok

Comment (wdpypere) [reply pakiti-client.spec]: my bad, it's fixed now.

**Pull request merged**